api/String.h

```diff
@@ -207,8 +207,8 @@ class String
 
 	// modification
 	void insert(char c, unsigned int index) { return insert(&c, index, 1); };
-    void insert(const String &str, unsigned int index, unsigned int length);
-    void replace(char find, char replace);
+    	void insert(const String &str, unsigned int index, unsigned int length);
```

Comment:
Not fixed very well, lines around uses single TAB and here four spaces and TAB -> for someone with tab displayed as 4 spaces it'll be broken (author obviously uses tab displayed as 8 spaces)

Reply:
Fixed. Let me know if you find more formatting issues